src/Models/Assessment/Excel/CoreReport/Financial_Summary.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
```

Comment:
P1: System is not being used in this class anywhere, it can be removed, as this would save execution time as the runtime environment won't need to unfold unnecessary classes.